index.html

```diff
@@ -0,0 +1,260 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="style.css">
+        <title>Capstone</title>
+    </head>
+    <body class="bodyart">
+        <header id="header" class="header">
+            <nav class="navbar">
+                <ul class="nav-list">
+                    <li>
+                        <img src="./images/Rectangle.png" class="facebook" alt="facebook"/>
+                    </li>
+                    <li>
+                        <img src="./images/twitter.png" class="facebook" alt="facebook"/>
+                    </li>
+                    <li>
+                        <a class="header-styles" href="#">English</a>
+                    </li>
+                    <li>
+                        <a class="header-styles" href="#">My Page</a>
+                    </li>
+                    <li>
+                        <a class="header-styles" href="#">Logout</a>
```

Comment:
Hi @Munsa1.

A coder review before, wrote that I can change this part and is the only par that wa can change. However, I made changes but, I invite to you to read before to do a code review because you confuse people with your comments.

I made changes as your solicitude now:

![image](https://github.com/jovasal/Capstone/assets/35693330/500dd874-ad5e-4b24-b825-d6d5dac7f980)
![image](https://github.com/jovasal/Capstone/assets/35693330/fc544547-a93e-42b9-a18a-780ff9b68515)

Reply:
The changes you have made are great, however to match the given design, the hieght of the image should have been longer e.g

Given design
![image](https://github.com/jovasal/Capstone/assets/81932478/9d26ee48-9bdf-4ded-80e8-fc3078bcf33b)

- Notice how the text to the right does not overflow beyond the bottom of the speaker image?